app/(app)/parking-cards/verification/index.tsx

```diff
@@ -31,7 +31,7 @@ const Page = () => {
     onSuccess: (res) => {
       const tmpVerificationToken = res.data[0].token
 
-      router.push({
+      router.replace({
```

Comment:
cannot be done like this, because if user wants to go back from verification-sent to email it wont be possible, just leave it with `push` and I will adjust new part of code I added to verification-result page :)

Reply:
done here: https://github.com/bratislava/paas-mpa/pull/340